src/extension/gogoduck/doc/wps-acorn.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Well, I reckon just keep one.   Should be easy enough given one working example for people to adapt to whatever they need?